geth/api/api.go

```diff
@@ -195,29 +190,17 @@ func (api *StatusAPI) JailBaseJS(js string) {
 	api.b.jailManager.BaseJS(js)
 }
 
-// TODO(oskarth): API package this stuff
+// Notify and send message.
 func (api *StatusAPI) Notify(token string) string {
 	log.Debug("Notify", "token", token)
 
-	var NP fcm.NotificationPayload
-	NP.Title = "Status - new message"
-	NP.Body = "ping"
-
 	// TODO(oskarth): Experiment with this
-	data := map[string]string{
+	msg := map[string]string{
 		"msg": "Hello World1",
 		"sum": "Happy Day",
 	}
```

Comment:
Just out of curiosity. How is this going to be unmarshalled on the receiving side and where is this information shown? I've seen a notification payload below. So what is this?
cc @oskarth

Reply:
@tiabc @JekaMas There's a notification payload which is what is visible to end user, then there is a data payload (this) which isn't visible but can be used. Then there are some additional settings such as CLICK_ACTION for Android.

Btw, we can extend the interface to be more generic (expose setting for other client, notification, data payload, CLICK_ACTION and similar things) if you guys want. Perhaps this is better done in a separate commit though. If you want to go down that route, we can make the necessary changes in status-react at the same time just so we have that capability, while keeping the same behavior as we have now.

Reply:
See https://github.com/status-im/status-react/issues/1883 and https://github.com/status-im/status-react/issues/1907 for some context from status-react side

Reply:
Thanks for the explanation and some related background, @oskarth!
What did you mean about our desire to make it more generic? If there's a requirements for that, we'd be happy to make it more general, just file an issue for that :)

Reply:
Actually i`ve make payload on our side: https://github.com/status-im/status-go/blob/028206653571379c3f7d6e2ce115c6217b02f80d/geth/notification/payload.go

Reply:
@tiabc No requirement right now.